locales/locales.php

```diff
@@ -270,8 +270,8 @@ public function __construct() {
 		$az->facebook_locale = 'az_AZ';
 
 		$azb = new GP_Locale();
-		$azb->english_name = 'South Azerbaijani';
-		$azb->native_name = 'گؤنئی آذربایجان';
+		$azb->english_name = 'Turki';
+		$azb->native_name = 'تورکجه (تورکی)';
```

Comment:
```suggestion
		$azb->english_name = 'South Azerbaijani';
		$azb->native_name = 'تۆرکجه‌ (آذربایجان تۆرکجه‌سی)';
```

Reply:
Not agree with changes. Explained in [#1762](https://github.com/GlotPress/GlotPress/issues/1762)